test/submission.js

```diff
@@ -4999,4 +4999,145 @@ module.exports = function(app, template, hook) {
         });
     });
   });
+
+  describe('Conditional Nested Forms Submissions', function () {
+    it('Sets up a default project', function (done) {
```

Comment:
All setup steps should be done in a before or beforeAll statement. This allows you to run any test in isolation.

```suggestion
    before('Sets up a default project', function (done) {
```

Reply:
changed